File: practice_algos/javascript/algo04.js
// 4. Iterate an array - Write a function that returns the sum of all the
// values within an array. (e.g. [1,2,5] returns 8. [-5,2,5,12] returns 14)

function iterateArray(arr) {
    var sum = 0;
    for (var i = 0; i < arr.length; i++) {
        sum += arr[i]
    }
    return sum;
}

console.log(iterateArray([1,2,5]))
console.log(iterateArray([-5,2,5,12]))

File: practice_algos/python/algo04.py
# 4. Iterate an array - Write a function that returns the sum of all the
# values within an array. (e.g. [1,2,5] returns 8. [-5,2,5,12] returns 14)

def iterate_array(arr):
    sum = 0
    for num in arr:
        sum += num
    return sum

print(iterate_array([1, 2, 5]))
print(iterate_array([-5,2,5,12]))